### ISSUE TITLE
Contracts that use libraries are not sized accurately

### ISSUE THREAD

**Issue by cgewecke** [ItsNickBarry/hardhat-contract-sizer#1]
Hi @ItsNickBarry! Love this plugin 

When sizing a contract whose bytecode includes library address placeholders, e.g. `__$....$__`, it seems like the size measurement is incorrect - it only returns size up to the first placeholder entry. 

A repro based on how length is measured here is:
```js
const a = "aaaa__$aaaa$__aaaa";
const b = "bbbbbbbbbbbbbbbbbb";

a_bufferLength = Buffer.from(a, 'hex').length;  // Issue w/ non-hex chars?
b_bufferLength = Buffer.from(b, 'hex').length;

a_stringLength = a.length;
b_stringLength = b.length;

console.log(`a_bufferLength: ${a_bufferLength}`);
console.log(`b_bufferLength: ${b_bufferLength}`);
console.log(`a_stringLength: ${a_stringLength}`);
console.log(`b_stringLength: ${b_stringLength}`);

> a_bufferLength: 2  // Short
> b_bufferLength: 9  // OK
> a_stringLength: 18
> b_stringLength: 18
```

**Comment by ItsNickBarry** [comment]
Does #2 correctly resolve this?  I'm not especially familiar with libraries.

**Comment by cgewecke** [comment]
Yeah! Looks good. I swapped that change into my local node_modules copy and got what I was expecting.

**Comment by ItsNickBarry** [comment]
Fixed in version `2.0.3`.
